state/machine_linklayerdevices.go

```diff
@@ -859,9 +859,86 @@ func (m *Machine) AllNetworkAddresses() ([]network.Address, error) {
 	for i := range stateAddresses {
 		networkAddresses[i] = stateAddresses[i].NetworkAddress()
 	}
+	// TODO(jam): 20161130 NetworkAddress object has a SpaceName attribute.
+	// However, we are not filling in that information here.
 	return networkAddresses, nil
 }
 
+
+// Should this be NetworkInterface rather than NetworkInfo?
+type NetworkInfo struct {
+	// If the machine is in the given space, it should have a primary host
+	// device that is in that space, which should be bridged.
+	HostDevice *LinkLayerDevice
+	BridgeDevice *LinkLayerDevice
+}
+
+// BridgeDevicesForSpaces takes a list of spaces, and returns the associated
+// bridge that should be used for that space. If no bridge has been created on
```

Comment:
Fixed. Now it is LinkLayerDevicesForSpaces, and just returns a list for each space.